drivers/usb/typec/tcpm/fusb302.c

```diff
@@ -709,6 +710,32 @@ static int tcpm_set_cc(struct tcpc_dev *dev, enum typec_cc_status cc)
 	return ret;
 }
 
+/*	For a single group of USB3.0, the signal direction
+	output of USB3.0 needs to be selected according to
+	the direction of TYPE C positive and negative insertion */
+static int fusb302_sel_gpio(struct fusb302_chip *chip){
+	if (!chip->gpio_sel || IS_ERR(chip->gpio_sel)) {
```

Comment:
```suggestion
	if (!chip || IS_ERR(chip->gpio_sel)) {
```
gpiod_set_value本身会检查输入是否为空